assets/dev/js/editor/utils/controls-css-parser.js

```diff
@@ -37,6 +37,34 @@ ControlsCSSParser = elementorModules.ViewModule.extend( {
 		} );
 	},
 
+	updateBackgroundImagesDimension( control, values ) {
+		const backgroundDimensionsPrefix = 'background_dimensions';
+		if ( control.name.includes( 'background_image' ) ) {
```

Comment:
Also, instead of a large `if` statement, to make code more readable and avoid nesting, I would just `return values` if the control isn't a background image control

Reply:
Not relevant anymore, the logic has changed.